### PR TITLE
feat: connect with stripe

### PR DESCRIPTION
**Pull request by LeeRenJie**

### CODE REVIEW

File: photo-app/config/initializers/stripe.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
+  :publishable_key => ENV['PUBLISHABLE_KEY'],
+  :secret_key      => ENV['SECRET_KEY']
```
Comment (tlkiong) [inline photo-app/config/initializers/stripe.rb:3]: How did you set these env ?

Comment (LeeRenJie) [reply photo-app/config/initializers/stripe.rb:3]: exported from a `.zshrc` file. I added to .gitignore

Comment (tlkiong) [reply photo-app/config/initializers/stripe.rb:3]: Thats where you set the env.
Now, think. If you are on a team, how would everyone knows what are the new env vars that you have (or add) in your project?

Comment (LeeRenJie) [reply photo-app/config/initializers/stripe.rb:3]: 🤔No idea, I searched around only found libraries or 3rd party services to share passwords/credentials within the team. What would be your recommendation?

File: photo-app/Gemfile [hunk 2]
```diff
@@ -34,6 +35,7 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'dotenv-rails'
```
Comment (tlkiong) [inline photo-app/Gemfile:38]: Why you need this in test?

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: photo-app/Gemfile (hunk 2 above)
Comment (LeeRenJie) [reply photo-app/Gemfile:38]: i was thinking if i test my app then i need the gem to read my `.env` files

Comment (tlkiong) [reply photo-app/Gemfile:38]: When you write test, you need to think of it in terms of it being run in an automated way. means, whatever automated way will already inject the env variables for you. Just like in production.

This gem is only required in development

**Pull request merged**